docs/install.md

```diff
@@ -383,3 +383,50 @@ agent = Agent(model)
 To use [Ollama](https://ollama.com/), you must first download the Ollama client, and then download a model.
 
 You must also ensure the Ollama server is running when trying to make requests to it. For more information, please see the [Ollama documentation](https://github.com/ollama/ollama/tree/main/docs)
+
+### SambaNova
```

Comment:
You may also need to add Sambanova LLM API to the list at the top of this document: 




> This installs the `pydantic_ai` package, core dependencies, and libraries required to use the following LLM APIs:
* [OpenAI API](https://platform.openai.com/docs/overview)
* [Google VertexAI API](https://cloud.google.com/vertex-ai) for Gemini models
* [Anthropic API](https://docs.anthropic.com/en/api/getting-started)
* [Groq API](https://console.groq.com/docs/overview)
* [SambaNova API](https://cloud.sambanova.ai/apis)